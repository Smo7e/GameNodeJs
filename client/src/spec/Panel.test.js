import React from 'react';
import { render } from '@testing-library/react';
import Panel from '../component/Lobby/panel/Panel';


test('Panel snapshot', () => {
    const { asFragment } = render(<Panel />);
    expect(asFragment()).toMatchSnapshot();
  });
