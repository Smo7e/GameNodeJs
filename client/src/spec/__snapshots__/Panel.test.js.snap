// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="panel"
    />
    <div
      class="panel"
    />
    <div
      class="panel"
    />
    <div
      class="panel"
    />
    <div
      class="panel"
    />
    <div
      class="panel"
    />
  </div>
</DocumentFragment>
`;
